dask_conversions/done/Dask_NDVI_Anomaly.ipynb

```diff
@@ -24,135 +24,50 @@
    "cell_type": "code",
```

Comment:
<p>uncomment one of pip install for first run-through</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SatelliteApplicationsCatapult/odc-hub/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>